chart/Chart.yaml

```diff
@@ -3,6 +3,7 @@ name: rancher
 description: Install Rancher Server to manage Kubernetes clusters across providers.
 version: %VERSION%
 appVersion: %APP_VERSION%
+kubeVersion: >= v1.17.17 < 1.22.0
```

Comment:
I don't think we typically add min version based on our support matrix. It will fail if user is upgrading from < k8s 1.17. So probably `< 1.21.0-0` should be enough. cc @deniseschannon @cbron

Reply:
+1, was just about to suggest the same. Now, do we need `< 1.22.0-0` or would `< 1.22.0` be enough?

Reply:
I think `< 1.22.0-0` exclude any pre-release too.

Reply:
done

Reply:
@StrongMonkey , just tested it a bit to be sure. `< 1.22.0-0` will actually allow pre-releases from 1.21 and below while `< 1.22.0` will not allow any pre-releases period.
So I think `< 1.22.0-0` is a better fit as it's less restrictive.

![image](https://user-images.githubusercontent.com/85187633/127206364-c3ac2eee-4c3d-4f0e-92b6-e6032deb03dc.png)